HeterogeneousCore/AlpakaMath/interface/Vector.h

```diff
@@ -0,0 +1,301 @@
+#ifndef HeterogeneousCore_AlpakaMath_interface_Vector_h
+#define HeterogeneousCore_AlpakaMath_interface_Vector_h
+
+// Modified and extended version based on VecArray (author: Felice Pantaleo, CERN)
+// Author: Alexei Strelchenko, FNAL
+//
+
+#include <utility>
+#include <type_traits>
+
+#include <alpaka/alpaka.hpp>
+
+#include "FWCore/Utilities/interface/CMSUnrollLoop.h"
```

Comment:
This `#include` requires adding
```xml
<use name="FWCore/Utilities" source_only="1"/>
```
to `HeterogeneousCore/AlpakaMath/BuildFile.xml`.